src/app.js

```diff
@@ -1 +1,32 @@
+/* eslint-disable no-console */
 'use strict';
+
+const path = require('path');
+const { copyFile } = require('./copyFile');
+
+const cp = () => {
+  const [source, dest] = process.argv.slice(2);
+
+  if (!source || !dest) {
+    throw new Error(
+      'You have to pass source and dest files path'
+    );
+  }
+
+  const sourcePath = path.isAbsolute(source)
+    ? source
+    : path.join(__dirname, source);
+  const destPath = path.isAbsolute(dest) ? dest : path.join(__dirname, dest);
+
+  if (sourcePath === destPath) {
+    console.log(
```

Comment:
Instead of console.log, you might want to throw an error, or at least return a message from the function to make it easier to handle errors and debug.